examples/fiber_configuration.rb

```diff
@@ -0,0 +1,58 @@
+# Example configuration for Gouda fiber-based job processing
+# This shows how to properly configure Rails and Gouda for optimal fiber performance
+
+# CRITICAL: Set Rails isolation level to :fiber (config/application.rb)
+# This MUST be set when using PostgreSQL to prevent segfaults and ensure proper fiber concurrency
+#
+# # config/application.rb
+# class Application < Rails::Application
+#   # Required for fiber mode with PostgreSQL - prevents segfaults and connection issues
+#   config.active_support.isolation_level = :fiber
+# end
+
+# Gouda fiber configuration
+Gouda.configure do |config|
+  # Enable fiber-based worker (instead of threads)
+  config.use_fiber_scheduler = true
+
+  # Number of concurrent worker fibers
+  # Can be higher than CPU cores since fibers are lightweight for IO-bound work
+  config.fiber_worker_count = 10
+
+  # Database connection pool size
+  # Should be >= fiber_worker_count + buffer for other Rails processes
+  config.async_db_pool_size = 25
```

Comment:
We have this value - but will Gouda actually reconnect to the database all by itself? Or does it only check this number is higher than `n_fibers` (or, once we combine - `n_threads * n_fibers`)? Because with the example here you connect to the DB without a pool, and I miss where Gouda manages the pool size by itself. I doubt we need this setting, to be honest.